package it.exceptionLearning.thro;

public class AETester {

	public static void main(String[] args) {
	

	}
	
	
	public static void generaEccezione () throws AlessandroException {
		
		throw new AlessandroException("si e' verificata un'eccezione");
	}

}
